client/connect-account-page/style.scss

```diff
@@ -91,7 +91,7 @@
 		}
 
 		.connect-account__card {
-			.woocommerce-card__body {
+			> .components-card__body {
```

Comment:
This needs to target direct descendant only because there is a gutenberg Card inside this card.